mods/d2k/rules/world.yaml

```diff
@@ -35,6 +35,10 @@
 		Name: Corrino
 		InternalName: corrino
 		Selectable: false
+	Faction@Smuggler:
```

Comment:
According to Tibed in orginal game Corrino has own CYard and War Factory. But they are same as Harkonnen ones as graphics and code. Possibly used in missions to separate them from Harkonnen ones.